zha/application/platforms/binary_sensor/__init__.py

```diff
@@ -197,12 +200,17 @@ class Motion(Opening):
 
     _attr_device_class: BinarySensorDeviceClass = BinarySensorDeviceClass.MOTION
 
+    # Many devices support both "motion" and "occupancy", with "motion" being the more
+    # useful entity to have
+    _attr_primary_weight = 3
+
 
 @MULTI_MATCH(cluster_handler_names=CLUSTER_HANDLER_ZONE)
 class IASZone(BinarySensor):
     """ZHA IAS BinarySensor."""
 
     _attribute_name = "zone_status"
+    _attr_primary_weight = 2
```

Comment:
This will be the motion entity for many sensors, like Aqara ones. It'll tie with the (fake) occupancy cluster we add.
I'd bump this to `_attr_primary_weight = 3`.